### PR TITLE
1010D Final Content Updates for User Testing (Part 2 of 2)

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

PART 2 OF 2

This PR updates the content and layout of all file upload components as well as reorganizes/refactors the main form config. It is a heavy refactor with a lot of code having been moved around, which contributes to the line counts being so high.

- Pull individual file upload uiSchemas out into a separate file
- Reorder pages pertaining to child education proof
- Content updates per CAIA feedback
- Team: IVC CHAMPVA
- Flipper: NA

## Testing done

- Manual testing
- Verified existing tests pass

## Screenshots

Example of updated file upload interface (one of several)

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |before1|after1|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/components/Applicant/applicantFileUpload.js [hunk 1]
```diff
@@ -1,76 +1,566 @@
+import React from 'react';
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/components/Applicant/applicantFileUpload.js:1]: This file now holds all the file upload UiSchemas along with their custom content

**Review by cloudmagic80 — DISMISSED** [review]

File: src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js [hunk 2]
```diff
@@ -190,48 +184,107 @@ export function uploadWithInfoComponent(
...
-      'view:fileUploadMessage': blankSchema,
+      // 'view:fileUploadMessage': blankSchema,
       'view:fileUploadBlurb': blankSchema,
     },
   };
 }
 
 export const sponsorDisabilityRatingConfig = uploadWithInfoComponent(
-  acceptableFiles.disabilityCert,
+  undefined, // acceptableFiles.disabilityCert,
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js:221]: Temporarily circumventing the acceptable files list functionality (you'll see this wherever `uploadWithInfoComponent` is used.

File: src/applications/ivc-champva/10-10D/config/form.js [hunk 3]
```diff
@@ -937,6 +902,67 @@ const formConfig = {
...
+          customPageUsesPagePerItemData: true,
+          uiSchema: applicantStepChildUploadUiSchema,
+          schema: applicantListSchema([], {
+            titleSchema,
+            ...applicantStepChildConfig.schema,
+            applicantStepMarriageCert: fileWithMetadataSchema(
+              acceptableFiles.stepCert,
+            ),
+          }),
+        },
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/config/form.js:965]: This is just a relocation of pages so they show up earlier in form flow

**Review by reiting — APPROVED** [review]

**Review by balexandr — APPROVED** [review]
lgtm

**Review by cloudmagic80 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**